add markup-main and some styles

Artem-Yampolskiy opened the pull request.

Artem-Yampolskiy committed:
add markup-main and some styles

lunatiq-san commented on src/partials/main.html:
```diff
@@ -0,0 +1,12 @@
+<main>
+    <div class="main-conteiner">
```
Change class on `container`

lunatiq-san commented on src/sass/base/_conteiner.scss:
```diff
@@ -0,0 +1,26 @@
+.main-container {
```
Change class on `container`

lunatiq-san commented on src/sass/base/_conteiner.scss:
```diff
@@ -0,0 +1,26 @@
+  margin-left: auto;
+  margin-right: auto;
+
+  @media screen and (min-width: 320px) {
```
You should use not px when writing min-width. Add in file `utils/_breakpoints.scss` breakpoints

lunatiq-san replied:
Change `px` on `$mobile`

lunatiq-san commented on src/sass/base/_conteiner.scss:
```diff
@@ -0,0 +1,26 @@
+    padding-top: 78px;
+    padding-bottom: 60px;
+  }
+  @media screen and (min-width: 768px) {
```
Change `px` on `$tablet`

lunatiq-san replied:
And delete all padding-top/bottom in container

lunatiq-san commented on src/sass/base/_conteiner.scss:
```diff
@@ -0,0 +1,26 @@
+    padding-top: 40px;
+    padding-bottom: 60px;
+  }
+  @media screen and (min-width: 1280px) {
```
Change `px` on `$desktop`

lunatiq-san commented on src/sass/main.scss:
```diff
@@ -8,4 +8,6 @@
-@import './utils/fonts';
\ No newline at end of file
+@import './utils/fonts';
+@import './base/conteiner';
```
Change `conteiner` on `container`

Artem-Yampolskiy committed:
fixed bag & 1 conflict

Исправил баги по контейнеру, переименовал классы, убрал падинги, использовал переменные

lunatiq-san commented on src/sass/utils/_breakpoints.scss:
```diff
@@ -1,5 +1,5 @@
-// $tablet-max:
-// $desktop:
+$mobile: 320px;
+$mobile-max: 319px;
```
Mobile max can equals `767px`

Artem-Yampolskiy replied:
Fixed

lunatiq-san commented on src/sass/utils/_breakpoints.scss:
```diff
@@ -1,5 +1,5 @@
+$mobile: 320px;
+$mobile-max: 319px;
+$tablet: 768px;
+$tablet-max: 767px;
```
Tablet max can equals `1279px`

Artem-Yampolskiy replied:
Fixed

lunatiq-san commented on src/partials/main.html:
```diff
@@ -0,0 +1,12 @@
+<main>
```
Посмотри на примере работы по HTML, CSS как в мейне вкладывается контейнер

Artem-Yampolskiy replied:
контейнеры вложить в секции ? :)

lunatiq-san replied:
Да, контейнер идет в каждой секции отдельно

Artem-Yampolskiy replied:
Fixed

Artem-Yampolskiy committed:
Fixed

Fixed container & breakpoints

lunatiq-san committed:
Merge branch 'dev' into feature/card-container-markup-&-styles
